### DISCUSSION TITLE
Update from v6.02 build 7961 to v6.08 build 8409

### DISCUSSION THREAD

**Discussion by kaeser49** [snipe/snipe-it#11556] [General]
Hi,

after updating to v6.08 to latest build without errors, Snipe-it started normally and did not show errors.

Unfortunately, the People Tab shows "no matching records found" and when I want to create a new user I get a server 500 error.

The existing people are still there and connected to their assets. I can even open link to the people.
All seems to work, except showing and creating people under the menu tab.
Can it be, that db columns are missing? I must say, that we imported all data from Excel to Snipe-IT v6.02 by CVS and made all id links by ourselves. As I cannot create a new user, I can not see what has changed, but I guess, a field or two are missing now.

Thank you for your support.

Werner

**Comment by kaeser49** [comment]
[2022-07-24 06:01:20] production.ERROR: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'users.created_by'

**Comment by kaeser49** [comment]
![image](https://user-images.githubusercontent.com/38448090/180634920-48c6fe88-305c-4b0d-92c1-7b672cdf32b7.png)

This made the trick. Please let me know if the field is correctly built. At least it works and I can create new users

**Comment by snipe** [comment]
You always have to run migrations when updating. If you used the upgrade.php, it will try to run those for you, but if it failed somewhere, we'd need to know the output of that upgrade script.

**Comment by kaeser49** [comment]
the final answer was: all good, enjoy Snipe-IT.
The recommendation was, to clear the browser, what was not needed and certainly, it would be a bad joke, to lose all stored browser data just to use an updated app.

Even though, we enjoy the software Snipe-IT very much. There are just a few points, where we believe, that the development has agreed the last years as being good inputs from the community, but did not find a way inside the tool.
Some of them are 
- customized date fields (some websites work already, some not, most people on earths are not US, believe it 2022.12.31 what?)
- Individual fields not only in assets, but in persons and other places too
- Possibility to set up own menu on the left pane. The developers should work with the tool every day in praxis to find out, that searching 
  often used menu points in the background isn't funny. i.E localization or deployable assets. 

Again: This software is awesome, accurate, fast and the search functionality is excellent. Thank you for that product.
